cmd/monitor/main.go

```diff
@@ -0,0 +1,55 @@
+// Service to monitor Adafruit feed uptime and temperatures.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/liftM/fishmon/pkg/adafruitio"
+	"github.com/liftM/fishmon/pkg/ds18b20"
+)
+
+func main() {
+	// Set up command-line flags.
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), `%s starts the feed monitoring service..
+
+Usage of %s:
+`, os.Args[0], os.Args[0])
+		flag.PrintDefaults()
+	}
+	aioUser := flag.String("aio_username", "", "Adafruit.IO username")
+	aioKey := flag.String("aio_key", "", "Adafruit.IO key")
+	var minTemp, maxTemp float64
+	flag.Float64Var(&minTemp, "min_temp", 65.0, "Lowest temperature allowed before alerting, in degrees Fahrenheit. Default is 65.0.")
```

Comment:
I prefer `flag.Type` over `flag.TypeVar`. I don't have a strong argument either way, but existing code uses `flag.Type` so we should remain consistent.